nulib/dc-nextjs
# Remove SSR and wrap embedded viewer with Auth

## What does this do?
- Removes Server Side Rendering from the app
- Reduces Static build count for Work and Collection from 100 to 1
- Embedded Viewer route will now respect auth.

## How to test
1. Click into a few Works and Collections and make sure they render logged in / not logged in.
2. View a Private, Published Work in the embedded viewer route.  To do this:
  - Go to a Private, Published Work, click Download and Share and see the iFrame embed code.  
  - Copy the `iframe` `src` attribute value, and paste into a new browser tab to view the Embedded Viewer
  - Try this logged in and logged out, and notice it will respect auth status.
